feed.xml

```diff
@@ -16,6 +16,34 @@
         <itunes:image href="https://remotehack.space/live/assets/big-hack.jpg" />
         <itunes:category text="Technology"/>
 
+
+        <item>
+            <title>Episode 19 - Remote Hack Macroterview</title>
```

Comment:
let's update this to have a very obvious "HackChat" somewhere in the title (maybe in brackets at the end?)

`Episode 19 - Ben and Ryan (HackChat)`
?

Reply:
great :+1: